### DISCUSSION TITLE
Provide the possibility to add an in-memory secret provider in the secret store for testing

### DISCUSSION THREAD

**Discussion by stijnmoreels** [arcus-azure/arcus.security#256] [Ideas]
When people are trying to test their code, and want to check some (non-secret) values from being used by their code. It may be an option to include also an in-memory secret provider, possible in separate project and with the 'testing' namespace to make sure that people understand that this is for testing.

Maybe this is even something for the Arcus.Testing library instead.

**Comment by tomkerkhove** [comment]
Sounds good, but Arcus.Testing indeed?

**Reply by stijnmoreels** [reply]
Yes, for Arcus.Testing indeed, but ok if you're cool with this, we can create an issue there. Thanks!
